xarray/tests/test_dask.py

```diff
@@ -1035,6 +1035,13 @@ def test_unify_chunks_shallow_copy(obj, transform):
     assert_identical(obj, unified) and obj is not obj.unify_chunks()
 
 
+@pytest.mark.parametrize("obj", [make_da()])
+def test_auto_chunk_da(obj):
+    expected = obj.chunk("auto").data
+    actual = obj.data.rechunk("auto")
```

Comment:
this should be the other way around :wink:

the result of `dask`'s `rechunk` is what we expect, actual should be what `chunk` returns. That doesn't change the result of the tests, but it makes identifying the test subject a bit easier.

Reply:
aaaaah jeez haha 

noted